locale/zh-cn/docs/guides/diagnostics/memory/using-heap-profiler.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Memory Diagnostics - Using Heap Profiler
+layout: docs.hbs
+---
+
+# 使用内存堆剖析器
+
+为了调试一个与内存相关的问题，我们需要观察特定对象占有了多少内存空间，以及哪些对象占用
+了他们触发了垃圾回收。为了有效的调试，我们也需要知道连续时间下我们的变量是如何在内存里
+分配的。
+
+内存堆剖析器处于 V8 的顶部，能持续对内存分配进行快照。在这篇文章里我们将设计到内存的
+剖析，将使用：
+
+1. 时间轴的分配
+2. 采样内存堆剖析器信息
+
+不同于[使用堆快照][]中所涉及到的内容，使用实时堆剖析意在了解一段特定时间下内存分配
+情况。
+
+## 堆剖析器 - 时间轴的分配
+
+堆剖析器与堆采样分析器相似，不同处在于它会跟踪每次的内存分配状况，它的开销也就
+高于堆采样分析器，所以不建议在生产环境中使用。
+
+> 你可借助 [@mmarchini/observe][] 通过编码方式实现。
+
+### 怎么用堆剖析器？
+
+启动应用程序：
+
+```console
+node --inspect index.js
+```
+
+> `--inspect-brk` 对于脚本而言是较好的选项。
+
+在 Chrome 中连接开发工具实例，然后：
+
+* 选择 `memory` 选项卡
+* 选择 `Allocation instrumentation timeline`
+* 开始剖析
+
+![堆剖析器步骤 1][heap profiler tutorial 1]
+
+然后堆剖析就开始运行了，在此我们强烈建议您运行示例，这样便于确认内存相关的问题。
+拿本例子而言，我们将使用 `Apache Benchmark` 工具弄出应用程序中的负载。
+
+> 在这个示例中，我们假定堆剖析基于 Web 应用程序。
+
+```console
+$ ab -n 1000 -c 5 http://localhost:3000
+```
+
+当负载全部请求完毕之后，请按“停止”按钮。
+
+![堆剖析器步骤 2][heap profiler tutorial 2]
+
+然后针对内存分配情况看一下快照。
+
+![堆剖析器步骤 3][heap profiler tutorial 3]
+
+请查阅下列有助于你了解关于更多内存相关术语的[链接部分](#usefull-links) 。
+
+## 堆剖析的采样
+
+对堆剖析器的采样是在一定时间内持续跟踪内存份分配状况，以及后备内存。由于采样基于低
+开销进行，所以它可以用在生产环境。
+
+> 你可以借助 [`heap-profiler`][] 模块，通过编程方式实现此功能。
+
+### 如何对堆剖析进行采样？
+
+启动应用程序：
+
+```console
```

Comment:
is it `bash`?

Reply:
I'm afraid not, it's something like in Linux as a command line. And it's directly copied from the English version.
For bash, it looks like something here:
![1658540838945](https://user-images.githubusercontent.com/52018749/180585797-20784e4a-b638-450c-9c37-e06727411038.png)